sklearn/preprocessing/data.py

```diff
@@ -2028,7 +2028,11 @@ def fit(self, X, y=None):
 
         if y is not None:
             for i in range(len_data):
-                X_key = tuple(X.take(i, axis=0).take(inclusion_used).flatten())
+                X_key_1 = X.take(i, axis=0)
```

Comment:
Temporary fix to try and hopefully debug 
```
ERROR: /home/travis/build/scikit-learn/scikit-learn/sklearn/tests/test_common.py.test_non_meta_estimators:check_fit_score_takes_y(CountFeaturizer)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/scikit-learn/scikit-learn/testvenv/local/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/travis/build/scikit-learn/scikit-learn/sklearn/utils/testing.py", line 741, in __call__
    return self.check(*args, **kwargs)
  File "/home/travis/build/scikit-learn/scikit-learn/sklearn/utils/testing.py", line 292, in wrapper
    return fn(*args, **kwargs)
  File "/home/travis/build/scikit-learn/scikit-learn/sklearn/utils/estimator_checks.py", line 736, in check_fit_score_takes_y
    func(X, y)
  File "/home/travis/build/scikit-learn/scikit-learn/sklearn/preprocessing/data.py", line 2031, in fit
    X_key = tuple(X.take(i, axis=0).take(inclusion_used).flatten())
ValueError: object of too small depth for desired array
```
Since I was unable to find anything relevant in the numpy source code or Google or stackoverflow  and I was unable to reproduce these errors on my own computer